app.js

```diff
@@ -0,0 +1,52 @@
+//Buttons Selectors
+const incrementBtn = document.querySelectorAll('.plus');
+const decrementBtn= document.querySelectorAll('.minus');
+const favBtn = document.querySelectorAll('.favorite');
+const deleteBtn = document.querySelectorAll('.remove');
+
+//Chnaging Values Selectors
+const productQuantity = document.querySelectorAll('.number');
+const subtotalAmount = document.querySelector('.subtotal-amount');
+const totalAmount = document.querySelector('.total-amount');
+const delivery = document.querySelector('.shipping-amount').innerText.slice(1);
+const pricesArr = document.querySelectorAll('.cart-item-price');
+let total = 0;
+//logic for the remove button
+for (let i = 0; i < deleteBtn.length; i++){
+    deleteBtn[i].addEventListener("click", (e)=>{
+        let btnClicked = e.target;
+        btnClicked.parentElement.remove();
```

Comment:
After deleting a product from the cart, the summary does not update. You should do some logic here to update the cart after removing a product.